session.go

```diff
@@ -57,7 +62,7 @@ func (ses *Session) Stream(ctx context.Context, in <-chan []cid.Cid) <-chan bloc
 		}
 	}()
 
-	return s.Output()
+	return s.Output(), ses.cherr
```

Comment:
We can't return the same error chan to all streams as there is only one possible read. Hence, we need to have a separate chan for every stream